mtwitter.js

```diff
@@ -0,0 +1,254 @@
+var Twitter = require('twitter');
+var Request = require('request');
+var fs = require('fs');
+var Buffer = require('buffer').Buffer;
+
+var TWITTER_PROFILE_INTERVAL = 60000;
+
+var mtwitter = function(bridge, config) {
```

Comment:
It bugs me that this isn't capitalised. It's a `Class`, but it's not styled as such.